tests/end2end/inprocess/test.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# inprocess test: TODO Add description of test.
+
+source ../utils.sh
+
+set -e
+
+test_init
+
+MODEL_SCRIPT=model.py
+MODEL_NAME=og_model.pt
+
+echo "Making model ${MODEL_NAME}"
+INIT_MODEL=`python ../${MODEL_SCRIPT} --action init --seed 1337 --model-name=${MODEL_NAME}`
+
+python test.py
+if [[ "$?" != 0 ]]; then
+    exit 1
+fi
+echo "Verifying that the same model saved in different paths match"
+python ../verify.py --old-model should_match_1.pt --new-model should_match_2.pt
```

Comment:
Is there a reason we can't use the functionality from verify.py directly in test.py rather than doing it separately in a shell script? Or is there a benefit to doing it separately?

Reply:
One reason is that verfiy.py isn't written in a way that makes importing it into other scripts easy, plus doing an import of a file in a different directory without having things setup as an installable package (__init__.py, setup.py, etc) can be weird.

Also the main runner for e2e tests is designed to run shell scripts so we would need to have a special case for this one and duplicate the color prints and things like that.